### DISCUSSION TITLE
Benchmarks appear to be missing

### DISCUSSION THREAD

**Discussion by mansu** [slackhq/kaldb#792] [General] [answered]
The benchmarking code is deleted from current master branch. While it's true that the benchmarks haven't kept pace with the rest of the code base, they are still a very valuable tool to quickly check the most used paths(read and write) in KalDB. Instead of deleting the benchmarks, can we update them instead? Some benchmarks are better than no benchmarks.

Without the benchmarks, we may be introducing a lot of regressions that would be extremely hard to catch otherwise.

**Comment by bryanlb** [comment, marked as answer]
I believe this question was addressed in the context of the original PR here https://github.com/slackhq/kaldb/pull/783/files#r1509346643. 

The existing benchmark code was exclusively written against Apilog transforms. This indexing path is no longer supported and is being removed, and the benchmarks will return once the indexer rework is complete.
